group.yml

```
@@ -1,4 +1,4 @@
-freeze_automation: false
+freeze_automation: true
```

that makes doozer refuse even manual builds. how about:
```suggestion
freeze_automation: scheduled
```

---

til, thanks!